src/bridge/lib/keys.js

```diff
@@ -22,26 +18,19 @@ const b64 = buf => {
     n = n.shrn(6)
   }
 
-  const trans = j =>
-      10 > j
-    ? j + 48
-    : 36 > j
-    ? j + 87
-    : 62 > j
-    ? j + 29
-    : 62 === j
-    ? 45
-    : 126
-
-  return '0w' +
-    c.reduce((a, b, i) =>
-      String.fromCharCode(trans(b)) +
-      (( i && 0 === i % 5 ) ? '.' : '') + a,
-      '')
+  const trans = j => (10 > j ? j + 48 : 36 > j ? j + 87 : 62 > j ? j + 29 : 62 === j ? 45 : 126)
```

Comment:
This seems kind of unfortunate. ):

Reply:
yikes, yeah that's pretty unreadable. we can ignore lines with `// prettier-ignore`, so in the last commit here I've reverted that specific line to what it was previously